lib/app.js

```diff
@@ -1,38 +1,10 @@
 const express = require('express');
-const Recipe = require('./models/recipe');
 const app = express();
 
 app.use(express.json());
 
-app.post('/api/v1/recipes', (req, res) => {
-  Recipe
-    .insert(req.body)
-    .then(recipe => res.send(recipe));
-});
-
-app.get('/api/v1/recipes', (req, res) => {
-  Recipe
-    .find()
-    .then(recipes => res.send(recipes));
-});
-
-app.get('/api/v1/recipes/:id', (req, res) => {
-  Recipe
-    .findById(req.params.id)
-    .then(recipe => res.send(recipe));
-});
-
-app.put('/api/v1/recipes/:id', (req, res) => {
-  Recipe
-    .update(req.params.id, req.body)
-    .then(recipe => res.send(recipe));
-});
-
-app.delete('/api/v1/recipes/:id', (req, res) => {
-  Recipe
-    .delete(req.params.id)
-    .then(recipe => res.send(recipe));
-});
+app.use('/api/v1/recipes', require('./controllers/recipe'));
+app.use('/api/v1/logs', require('./controllers/recipe'));
```

Comment:
Looks like you're requiring the wrong file here

Reply:
Got it, thank you!